spec/features/staff/users_can_create_a_project_level_activity_spec.rb

```diff
@@ -41,7 +42,7 @@
         expect(project.originating_report).to eq(report)
         expect(report.new_activities).to eq([project])
 
-        created_activity = Activity.order("created_at ASC").last
+        created_activity = form.created_activity
```

Comment:
Good catch, I skipped over those previous lines because I didn't think they were related to what I was doing, but yeah, we should use `created_activity` here